OpenRA.Mods.Common/Widgets/Logic/GlobalChatLogic.cs

```diff
@@ -38,9 +38,8 @@ public GlobalChatLogic(Widget widget)
 			inputBox.IsDisabled = () => Game.GlobalChat.ConnectionStatus != ChatConnectionStatus.Joined;
 			inputBox.OnEnterKey = EnterPressed;
 
-			// Set a random default nick
-			if (Game.Settings.Chat.Nickname == new ChatSettings().Nickname)
-				Game.Settings.Chat.Nickname += Game.CosmeticRandom.Next(100, 999);
+			if (string.IsNullOrEmpty(Game.Settings.Chat.Nickname))
```

Comment:
The global nickname will be player name (default also after changing player name), until user connects to global chat first time with his choosen name (either default player name or replaced by his nickname), then this nickname will be remembered.
We can remove Chat.Nickname, but it solves different request/issue - this PR ensures playername is used as default but still allows different nicknames.